machine-learning/app/models/facial_recognition.py

```diff
@@ -28,7 +28,10 @@ def __init__(
 
     def _load(self) -> None:
         self.det_model = RetinaFace(session=self._make_session(self.det_file))
-        self.rec_model = ArcFaceONNX(self.rec_file.as_posix(), session=self._make_session(self.rec_file))
+        self.rec_model = ArcFaceONNX(
+            self.rec_file.with_suffix(".onnx").as_posix(),
```

Comment:
Out of curiosity, what was the issue with passing the `.armnn` path here? Was `ArcFaceONNX` checking that it exists?

Reply:
It analyzes the onnx model to get the input dimensions etc. This requires a file parsable by an onnx parser^^